docs/eni-and-ip-target.md

```diff
@@ -0,0 +1,49 @@
+| t3.small      |         1         |         -        |          -          |   9  |        3       |            9            |      0     |    3,3,3    |
+|               |                   |                  |                     |      |                |                         |            |             |
+| t3.small      |         -         |         1        |          1          |   0  |        1       |            1            |      1     |      1      |
+| t3.small      |         -         |         1        |          1          |   5  |        2       |            6            |      1     |     3,3     |
+| t3.small      |         -         |         1        |          1          |   9  |        3       |            9            |      0     |    3,3,3    |
+|               |                   |                  |                     |      |                |                         |            |             |
+| t3.small      |         -         |         2        |          5          |   0  |        2       |            5            |      5     |     2,3     |
+| t3.small      |         -         |         2        |          5          |   5  |        2       |            7            |      2     |    3,3,1    |
+| t3.small      |         -         |         2        |          5          |   9  |        3       |            9            |      0     |    3,3,3    |
+|               |                   |                  |                     |      |                |                         |            |             |
+| p3dn.24xlarge |         1         |         -        |          -          |   0  |        1       |            49           |     49     |      49     |
+| p3dn.24xlarge |         1         |         -        |          -          |   3  |        2       |            98           |     95     |    49,49    |
+| p3dn.24xlarge |         1         |         -        |          -          |  95  |        3       |           147           |     52     |   49,49,49  |
+|               |                   |                  |                     |      |                |                         |            |             |
+| p3dn.24xlarge |         -         |         5        |          2          |   0  |        1       |            5            |      5     |      5      |
+| p3dn.24xlarge |         -         |         5        |          2          |   3  |        1       |            5            |      2     |      5      |
+| p3dn.24xlarge |         -         |         5        |          2          |  10  |        1       |            12           |      2     |      12     |
+|               |                   |                  |                     |      |                |                         |            |             |
+
+
+To be extra clear, only set `WARM_IP_TARGET` for small clusters, or clusters with very low pod churn. It's also advised 
```

Comment:
s/extra//

Something is either clear or not :)

Reply:
ESL! 😁 Thanks, addressed all your comments.